images/ad-server/provision.sh

```diff
@@ -2,8 +2,8 @@
 
 set -e
 
-REALM="ZZZ-BEST.X.ASYNCHRONO.US"
-DOM="ZZZ-BEST"
+REALM="samba1.sink.test"
```

Comment:
Unfortunately, I think using lower case here causes an error with the provisioning tool. You'd naturally think the domain is case sensitive but this [samba wiki page](https://wiki.samba.org/index.php/Setting_up_Samba_as_an_Active_Directory_Domain_Controller) says it must be upper cased.

Reply:
This was not the error. It worked before with `testdom`. The problem was that `samba1` is also the netbios name of the domain controller. Updated...